### ISSUE TITLE
Changing settings resets the Theme

### ISSUE THREAD

**Issue by MeFisto94** [Pavion/tvstreamrecord#65]
So, I was changing anything from the settings which lead me to having a basic theme out of a sudden. "smoothness" doesn't seem to be the default, though.

Also, when I look at the settings, I see this:
![grafik](https://github.com/user-attachments/assets/74a94d38-d13f-4f9f-8230-044540abab2e)

I'm on the most recent docker version (v1.6.4)

**Comment by Pavion** [comment]
Yeah, I know this bug and was never able to pinpoint it. You should still be able to select and append a theme anyway. Is it a breaking issue?
